cmake/Modules/PVS-Studio.cmake

```diff
@@ -0,0 +1,719 @@
+# 2006-2008 (c) Viva64.com Team 2008-2018 (c) OOO "Program Verification Systems"
```

Comment:
I accept this file as-is, we should probably exclude it from cmake-format somehow to make diffs easier to work with upon updates.

Reply:
I moved it to `cmake/external` and excluded that folder from formatting in `CI/check-cmake.sh`. I also added a note on the modifications made from upstream for it to work correctly here.